Tools/AP_Bootloader/board_types.txt

```diff
@@ -430,7 +430,10 @@ AP_HW_CUAV-7-NANO                    7000
 # IDs 7100-7109 reserved for V-UAV
 AP_HW_VUAV-V7pro                     7100
 
-# please fill gaps in the above ranges rather than adding past ID #7109
+# IDs 7200-7209 reserved for AEROFOX
+AP_HW_AEROFOX_H7                     7200
+
+# please fill gaps in the above ranges rather than adding past ID #7209
```

Comment:
Why did you not follow these instructions?

Reply:
Sorry, I will fix these soon